config/val.config.src

```diff
@@ -57,7 +57,9 @@
    %% we only really need the params below if this file is changed to specify a radio device
    %% as without one miner_lora is not started
    %% including the params anyway in case someone needs it in this env
-   {region_override, 'US915'}
+   {region_override, 'US915'},
+   %% max exponential backoff to apply to grpc reconnect
+   {max_backoff, 128}
```

Comment:
I think we discussed making this up to 10 minutes?

Reply:
this is a multiplier to the default backoff of 5 secs, so 5 * 128 = 640 secs or just over 10 mins.  
Updated comment to explain better and also moved this config to sys config as its a clientside config not validator
ca29e9f79c837386f36ed16d1f32c9fcc37cc283